pages/exercises/[lessonSlug].tsx

```diff
@@ -20,6 +19,13 @@ import ExerciseCard, { Message } from '../../components/ExerciseCard'
 import { ArrowLeftIcon } from '@primer/octicons-react'
 import GET_EXERCISES from '../../graphql/queries/getExercises'
 import styles from '../../scss/exercises.module.scss'
+import LessonTabs from '../../components/LessonTabs'
+
+type ExerciseLesson = {
+  title: string
+  docUrl?: string | null
```

Comment:
Why is there `null` here as a possible option?

Reply:
[The lesson type for `docUrl` is `string` or `null`](https://github.com/flacial/c0d3-app/blob/0dd1d4dcb09ce7978f98501af1e449bdb56a8792/graphql/typeDefs.ts#L215)